tools/pkg.yaml

```diff
@@ -89,6 +89,7 @@ steps:
   - prepare:
     - |
       # clean up some files we don't ever need
+      ls -alh /usr/man /usr/share/doc /usr/share/info /usr/share/locale /usr/share/man
```

Comment:
testing for other packages installing mans

Reply:
needed?